dev/config/pyramid_oereb.yml.mako

```diff
@@ -88,6 +88,16 @@ pyramid_oereb:
     # more time to generate the PDF. If set to false, it will assume that only one TOC page exists, and this can
     # lead to wrong numbering in the TOC.
     compute_toc_pages: true
+    # To avoid the potentially time consuming second computing of the PDF extract and skip the the computation
+    # of the estimated TOC length, you can specify a default length for the number of TOC pages.
+    # For most of the cantons the length of the TOC is pretty consistent unless a real estate is concerned by none 
+    # or a huge number of restrictions.
+    # An additional page break might also occur if the number of published topics is close to a threshold number
+    # where the TOC fits just about on one or two pages. - for those case estimate the TOC length ist preferable.
+    # In both cases (computing an estimated length or setting a default length) the exact number of TOC pages is
+    # extracted from the created PDF and if it is different from the expected value the PDF extract is called a 
+    # second time with the correct page numbers.
+    default_toc_length: 2
```

Comment:
Maybe the name of the variable could be more precise. E.g. "most_frequent_toc_length"